mesa-quilt

```diff
@@ -17,18 +17,18 @@
 # Free Software Foundation, Inc.,
 # 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
-d=. ; while [ ! -d "$d/nx-X11/extras" ] && [ "$(readlink -e "$d")" != "/" ]; do d="$d/.."; done
-if [ -d "$d/nx-X11/extras" ] && [ -z "$QUILT_PATCHES" ]; then
-    # if in nx-libs tree with unset $QUILT_PATCHES
+d=. ; while [ ! -d $d/nx-X11/extras -a `readlink -e $d` != / ]; do d=$d/..; done
+if [ -d $d/nx-X11/extras ] && [ -z $QUILT_PATCHES ]; then
+    # if in Debian packaging tree with unset $QUILT_PATCHES
     export QUILT_PATCHES="../Mesa.patches"
     export QUILT_PATCH_OPTS="--reject-format=unified"
     export QUILT_DIFF_ARGS="-p ab --no-timestamps --no-index --color=auto"
     export QUILT_REFRESH_ARGS="-p ab --no-timestamps --no-index"
     export QUILT_COLORS="diff_hdr=1;32:diff_add=1;34:diff_rem=1;31:diff_hunk=1;33:diff_ctx=35:diff_cctx=33"
-    if ! [ -d "$d/nx-X11/extras/Mesa.patches" ]; then mkdir "$d/nx-X11/extras/Mesa.patches"; fi
-    cd "$d/nx-X11/extras/Mesa/"
+    if ! [ -d $d/nx-X11/extras/Mesa.patches ]; then mkdir $d/nx-X11/extras/Mesa.patches; fi
+    cd $d/nx-X11/extras/Mesa/
 fi
 
-quilt "$@"
+quilt $@
```

Comment:
Don't change the mesa-quilt script at all. The version you had on my old branch is outdated, the one in 3.6.x is better.